async.lua

```diff
@@ -0,0 +1,23 @@
+areas = {}
```

Comment:
Prevent overwriting when loaded without async support.
```suggestion
areas = rawget("_G", "areas") or {}
```